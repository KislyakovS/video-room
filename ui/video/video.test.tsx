import { render } from '@testing-library/react';

import Video from './video';

describe('Video Component:', () => {
    it('snapshot', () => {
        const { container } = render(
            <Video title='Зеленая миля' controls>
                <source src="./src-video" type="video/mp4" />
            </Video>
        );

        expect(container).toMatchSnapshot();
    })
});