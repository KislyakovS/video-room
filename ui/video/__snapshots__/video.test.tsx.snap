// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Video> snapshot 1`] = `
<div>
  <section
    controls=""
  >
    <h3
      class="title"
    >
      Зеленая миля
    </h3>
    <video
      class="video"
      controls=""
    >
      <source
        src="./src-video"
        type="video/mp4"
      />
    </video>
  </section>
</div>
`;
